Advanced-Recruit-Training/README.adoc

```diff
@@ -0,0 +1,52 @@
+= U of C Solar Car Advanced Recruit Training
+Intro project to learn c++ and QT
+
+== Running the project
+
+=== Via Command Line
+
+To build this project on linux navigate to the source directory and run the following commands:
+
+`qmake`
+
+`make`
+
+`./batteryLife` - The command line arguments are as follows:
```

Comment:
I would put all 3 commands in one block.

[source,bash]
----
qmake
make
./batteryLife -f myData.csv -i 50
----

Also earlier I meant to just add those arguments up top ^, the whole explanations below were not necessary, like you said they're in the other README, I just thought it was wrong to give a command that wasn't actually how it's used.